Bugfix/game jam

m4rtint opened the pull request:
Mostly controller and map fixes

m4rtint commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Game Jam bugs](https://trello.com/c/4XMKAYpU/99-game-jam-bugs)

mikeslgoh commented on Assets/InControl/Source/VersionInfo.cs:
```diff
@@ -28,7 +28,7 @@ public static VersionInfo InControlVersion()
 				Minor = 4,
 				Patch = 4,
-				Build = 3956
+				Build = 3978
```
So what is this version info for?

m4rtint replied:
InControl does this thing, idk why. The file is needed locally. Technically can gitignore

mikeslgoh replied:
Yeah maybe we should add to gitIgnore coz it keeps climbing but no real reason for it besides InControl needing it.

m4rtint replied:
realized u can't gitignore it. 
If u clone it, u'lll need this file.
but possible to change this file so it doesn't keep changing

mikeslgoh replied:
hmm if it's quite a bit of work to go in and change it then leaving it is fine.

mikeslgoh commented on Assets/Prefabs/Parking/ParkingEight Variant.prefab:
```diff
@@ -870,7 +870,7 @@ PrefabInstance:
         type: 3}
       propertyPath: m_RootOrder
-      value: 7
+      value: 3
```
So did you find what was wrong with the parking spot?

m4rtint replied:
I haven't found the issue yet, i'm just reorganizing the spot for now.

m4rtint commented:
@mikeslgoh 
For the Controller. I removed the InControl GameObject from the map scene
dontDestroyOnLoad -> True on Main Menu
Removed InControl object from all the game maps scenes.

m4rtint commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Photoshop/Edit car indicator arrow](https://trello.com/c/fZ4cMwIp/88-photoshop-edit-car-indicator-arrow)

m4rtint commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Photoshop/Edit car indicator arrow](https://trello.com/c/fZ4cMwIp/88-photoshop-edit-car-indicator-arrow)

Gursaanj commented:
Could you add the Canvas with the Pause and End Game panel on the NPC Scenes, then all should be fine. Alternatively I could do that too

m4rtint commented:
@Gursaanj 
u can add changes to this branch with the pause/endgame panel on npc scenes if u want.if u haven't done it, i will do it

m4rtint commented:
@Gursaanj 
I placed the user interface panels onto the AI Scenes.
